lib/engine/game/g_18_esp/game.rb

```diff
@@ -917,6 +923,16 @@ def transfer_luxury_ability(company, entity)
           company.close!
         end
 
+        def transfer_tenders_to_bank
+          @luxury_carriages.dup.each do |owner, amount|
+            next if owner == 'bank'
+
+            transfer_amount = amount - 1
+            @luxury_carriages['bank'] += transfer_amount
+            @luxury_carriages[owner] = 1
```

Comment:
Is this correct? I do not know the code so I might be wrong. Feels strange to set this to one here. Just checking.

Reply:
Yes, there is a private that comes with 5 tenders, but tenders can end up in the "bank"
other corps can buy tenders either from player owner of the private or the bank.
If private is bought in, all the tenders are transferred the the bank, except one which is owned by the corp buying the private in.